R/pkg/R/DataFrame.R

```diff
@@ -411,7 +411,7 @@ setMethod("coltypes",
 #'
 #' Set the column types of a SparkDataFrame.
 #'
-#' @param x A SparkDataFrame
+#' @param y A SparkDataFrame whose column types to be set
```

Comment:
I see - it's common in R to get the getter/setter method to share the same document page, and event parameter lists, eg.
https://stat.ethz.ch/R-manual/R-devel/library/base/html/colnames.html

I think the best way to do that is not to rename it to `y`, but leave out the `@param x` for `coltype<-`